var/spack/repos/builtin/packages/superlu-dist/package.py

```diff
@@ -53,14 +53,16 @@ class SuperluDist(CMakePackage, CudaPackage, ROCmPackage):
         ),
     )
     variant("shared", default=True, description="Build shared libraries")
+    variant("parmetis", default=True, description="Build the ParMETIS library")
```

Comment:
Can you change the description to "Enable parmetis support" or something similar? Like this it seems the package is activating a vendored build, which is not the case.

Reply:
I copying the description string from the package CMakeLists.txt but this makes sense to me. What do you think @liuyangzhuan ?

Reply:
@jacobrking  Yes "Enable parmetis support" makes more sense. I will change it in CMakeLists.txt. Could you change it for spack?

Reply:
Just pushed these changes. Hope it is OK. If it is not please tell me and I won't do it again.

Reply:
Thank you for pushing the change. I'm at a conference this week so I'm slow to get to it.